src/python/gudhi/cover_complex.py

```diff
@@ -349,6 +361,8 @@ def fit(self, X, y=None, filters=None, colors=None):
                 filter functions (sometimes called lenses) used to compute the cover. Each column of the numpy array defines a scalar function defined on the input points.
             colors : list of lists or numpy array of shape (num_points) x (num_colors)
                 functions used to color the nodes of the cover complex. More specifically, coloring is done by computing the means of these functions on the subpopulations corresponding to each node. If None, first coordinate is used if input is point cloud, and eccentricity is used if input is distance matrix.
+            n_jobs :  The maximum number of concurrently running jobs (default 1). See https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html for details.
+            backend : Specify the parallelization backend implementation (default 'loky'). See https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html for details.
```

Comment:
In the rest of Gudhi, I don't think we expose this argument, users can use the parallel_config context manager to change the default. Is it recommended practice to expose this?